src/main/java/io/jenkins/plugins/PurgeBuildQueueAction.java

```diff
@@ -87,11 +82,11 @@ public String getDisplayName() {
 
     @Override
     public String getIconFileName() {
-        return Jenkins.get().hasPermission(Jenkins.ADMINISTER) ? "gear.svg" : null;
```

Comment:
Why has this permission check been removed? Now the icon is shown to everyone with Overall/Read permissions

Reply:
This needs to be done differently now.